### DISCUSSION TITLE
Can you help me out?

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [krishguptadev/krishguptadev#13] [General]
Hey,

Do you have any experience in HTML, JS?

**Comment by suitupstan** [comment]
Yeah, feel free to ask the question

**Reply by KendallDoesCoding** [reply]
I need help with my project, dropping the project url and help needed in  a bit. Thanks

**Reply by suitupstan** [reply]
@KendallDoesCoding can you send it?

**Reply by KendallDoesCoding** [reply]
Doing it rn lol

**Reply by KendallDoesCoding** [reply]
Added it in this thread, hope you recieved.

**Comment by KendallDoesCoding** [comment]
**The project is: [A Very Mogul Christmas!](mogulchristmas.kendalldoescoding.gq)**
**[Github Repo](https://github.com/KendallDoesCoding/mogul-christmas)**

Help Needed:

**HTML and CSS:**
[Move the Lyrics/Individual Song Page and Mogul Christmas Playlist button to the top right/left corner ](https://github.com/KendallDoesCoding/mogul-christmas/issues/106)
[Make site responsive and make it so that if screen size is small the button to toggle darkmode becomes small too.
](https://github.com/KendallDoesCoding/mogul-christmas/issues/80)

**JS:**
[[FEATURE REQUEST] Volume button to adjust volume
](https://github.com/KendallDoesCoding/mogul-christmas/issues/105)
[[FEATURE REQUEST] Loop song button](https://github.com/KendallDoesCoding/mogul-christmas/issues/187)
[Format lyrics proper in lyrics directory songs](https://github.com/KendallDoesCoding/mogul-christmas/issues/103)
[[FEATURE REQUEST] Remember if user wants darkmode or lightmode](https://github.com/KendallDoesCoding/mogul-christmas/issues/98)
[Make a directory to thank contributors on the site](https://github.com/KendallDoesCoding/mogul-christmas/issues/192)
[Update images from day/night to dark/light](https://github.com/KendallDoesCoding/mogul-christmas/issues/138)

If you don't understand a certain issue, feel free to reach out,

Kendall Does Coding,
contact@kendalldoescoding.gq

**Reply by suitupstan** [reply]
Wait, you don't need technical help but a contribution? 😅

**Reply by KendallDoesCoding** [reply]
Idk how to do it, so I need help, but its kinda a contribution, if your not down then it's ok lol

**Reply by suitupstan** [reply]
Which one do you don't know how to do?

**Reply by KendallDoesCoding** [reply]
All the ones in JS and responsive one in HTML, the first one in HTML, I know how to do but may need some inputs and basic help too.

**Comment by KendallDoesCoding** [comment]
If you can help me with this, I'll give you the required details. We have 2 contributors on the project already,

**Comment by KendallDoesCoding** [comment]
I'll drop you a email with this information, that's a better way of communicating than github discussions lol
